frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -152,9 +157,12 @@ describe("Where's My Gene", () => {
     await datasetOptions[0].click();
     await page.keyboard.press("Escape");
 
-    await clickUntilOptionsShowUp({ page, testId: ADD_TISSUE_ID });
-    const numberOfTissuesAfter = await countLocator(page.getByRole("option"));
-    await page.keyboard.press("Escape");
+    // wait for spinner to disappear, coincides with y-axis update
+    await waitForElementToBeRemoved(page, "loading-spinner");
```

Comment:
Is there a better way to know if the page has finished loading?

Reply:
`tryUntil` will retry the assertion for N times and then throw the assertion error. In this case, if `tryUntil` fails, it will show that `numberOfTissuesBefore` and `numberOfTissuesAfter` are not equal!